pkg/api/server/register_containers.go

```diff
@@ -1176,15 +1176,35 @@ func (s *APIServer) registerContainersHandlers(r *mux.Router) error {
 	//    description: the name or ID of the container
 	//  - in: query
 	//    name: condition
-	//    type: string
-	//    description: |
-	//      wait until container is to a given condition. default is stopped. valid conditions are:
-	//        - configured
-	//        - created
-	//        - exited
-	//        - paused
-	//        - running
-	//        - stopped
+	//    type: integer
+	//    enum:
+	//      - 1
+	//      - 2
+	//      - 3
+	//      - 4
+	//      - 5
+	//      - 6
+	//      - 7
+	//      - 8
+	//    x-enumNames:
```

Comment:
I added this NSwag extension property (it's not OpenAPI), but it will be useful for people using NSwag. Hope it's OK.

Reply:
I wish they have this in standard OpenAPI :disappointed: